### PR TITLE
bump: Bump codacy-sbt-plugin

### PR DESCRIPTION
**Pull request by lolgab**
- Use bundle publishing
- Refactor CI script
- Use coverage script
- Bump sbt to 1.3.8

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,20 +12,17 @@ workflows:
...
-            sbt "coverage;
-                 coverageReport;
-                 coverageAggregate"
+                 sbt:scalafmt::test;
+                 +test;
+                 coverage;
+                 coverageReport"
             if [ -z "$CODACY_PROJECT_TOKEN" ]; then
               echo "CODACY_PROJECT_TOKEN not found. Skipping send coverage to Codacy."
             else
```
Comment (josemiguelmelo) [inline .circleci/config.yml]: No need for the `if` statement. The coverage reporter does this check

Comment (lolgab) [reply .circleci/config.yml]: Right!

Comment (lolgab) [reply .circleci/config.yml]: Done ✅

**Comment by franciscodua** [comment]
Was the binary `.codacy-coverage/codacy-coverage-reporter-assembly.jar` added by mistake? Maybe we need to add it to .gitignore

**Pull request merged**